src/main/scala/no/gdl/bookapi/controller/OPDSController.scala

```diff
@@ -8,56 +8,131 @@
 package no.gdl.bookapi.controller
 
 import java.text.SimpleDateFormat
+import java.time.{LocalDate, ZoneId}
+import java.time.format.DateTimeFormatter
 import java.util.Date
+import javax.servlet.http.HttpServletRequest
 
-import no.gdl.bookapi.BookApiProperties.DefaultLanguage
-import no.gdl.bookapi.service.{ConverterService, ReadService}
+import no.gdl.bookapi.BookApiProperties
+import no.gdl.bookapi.model.api.{Feed, FeedEntry}
+import no.gdl.bookapi.service.{ConverterService, FeedService, ReadService}
+import org.scalatra.NotFound
 
 trait OPDSController {
-  this: ReadService with ConverterService =>
+  this: ReadService with ConverterService with FeedService =>
   val opdsController: OPDSController
   val sdf: SimpleDateFormat = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ssXXX")
+  val dtf: DateTimeFormatter = DateTimeFormatter.ISO_OFFSET_DATE_TIME
+  implicit val localDateOrdering: Ordering[LocalDate] = Ordering.by(_.toEpochDay)
 
   class OPDSController extends GdlController {
-
-    val page = 1
-    val pageSize = 10000 //TODO: Create partial opds feed entries, to solve paging
-
     before() {
       contentType = formats("xml")
     }
 
-    get("/:lang/catalog.atom") {
-      showOpdsForLanguage(params("lang"))
+    get(BookApiProperties.OpdsNavUrl.url) {
+      val navFeeds = feedService.feedsForNavigation(language("lang"))
+      val navUpdated = navFeeds.map(_.updated).sorted.reverse.headOption
```

Comment:
`.sorted.reverse` is not 100% intuitive, could this value be named `navLastUpdated` or something similar to more accurately show the intent?